integration/spark/build.gradle

```diff
@@ -191,7 +191,7 @@ sourceSets {
     spark3Test {
         java {
             srcDirs = ["src/test/common/java", "src/test/spark3/java"]
-            compileClasspath = configurations.spark3Test + sourceSets.main.output
+            compileClasspath = configurations.spark3Test + sourceSets.main.output + sourceSets.spark3.output
```

Comment:
This fixes code completion for Spark 3 tests in IntelliJ. I can make separate PR with that if needed.